apps/backend/src/utils/certificate.js

```diff
@@ -99,21 +122,248 @@ function getPlainTheme({
         </p>
         <div class="border-b border-gray-500 mb-4">
           <p class="text-xl text-black font-normal">This is to certify that</p>
-          <p class="student-name text-center text-8xl">${studentName}</p>
+          <p class=" text-center text-8xl student-name">${studentName}</p>
         </div>
         <div class="mb-2">
           <p class="text-xl text-black font-normal">
             has successfully completed a courses in
-          </p>
+          // </p>
           <div class="my-10">
             <p class="bg-transparent font-semibold text-pink-700 text-center uppercase text-3xl">
             ${courseName}
             </p>
           </div>
         </div>
         <p class="text-xl text-center text-gray-500 font-light">${courseDescription}</p>
+        <div class="flex justify-between w-[70%] mx-auto text-center items-end my-14">
+          <div>
+            <p class="font-bold">${dateIssued}</p>
+            <h1 class="border-t border-black h-2/4 font-bold">Date Issued</h1>
+          </div>
+
+          <div class="w-[20%]">
+            <h1 class="h-2/4 border-b border-black"></h1>
+            <p class="font-bold">Signature</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  `;
+
+  return getHtmlTemplate(themeBody);
+}
+
+function getPurpleProfessionalBadge({
+  studentName,
+  courseName,
+  courseDescription,
+  orgLogoUrl,
+  orgName,
+  facilitator,
+  dateIssued
+}) {
+  const themeBody = `
+    <img
+      src="http://localhost:3002/purple-professional-badge-background.png"
```

Comment:
Oh damin, I didn't notice this earlier.

You should hard code a URL to localhost in your code that will be deployed to production 😢 

Please delete the static folder cause I will upload the images to cloudflare and provide you the URL.

Then delete the code:

```
// serve the images in the static folder
app.use(express.static('static'));
```

For this one use: https://assets.cdn.clsrio.com/certificates/purple-professional-badge-background.png

Then replace all the templates you have used localhost.